src/Native/build.proj

```diff
@@ -78,9 +78,9 @@
     </ItemGroup>
     <ItemGroup Condition="'$(UseIntrinsics)' == 'true'">
       <NativePackageAsset Include="$(NativeAssetsBuiltPath)\$(NativeLibPrefix)CpuMathNative$(NativeLibExtension)"
-                          RelativePath="Microsoft.ML.CpuMath\runtimes\$(PackageRid)\lib\netstandard2.0" />
+                          RelativePath="Microsoft.ML.CpuMath\runtimes\$(PackageRid)\nativeassets\netstandard2.0" />
```

Comment:
Shouldn't nativeassets be under `x86`, `x64`, or `arm` folders (rather than `netstandard2.0`?

Reply:
The CpuMathNative native binaries are under both `RID` and `TFM` folders. Both `RID` and `TFM` can be used to decide which asset to be picked. NuGet supports pivoting on both of these properties.

`runtimes\$(Rid)\nativeassets\$(Tfm)`

The idea is that on `netcoreapp3.0`, we won't even need this native binary, because it will no longer be called. All of the logic in it will be replaced with C# intrinsic methods.

So, when running on a non-netcoreapp3.0 platform (like .NET Core 2.1, or netfx), use the CpuMathNative native binary for the appropriate `RID`.  But when running on `netcoreapp3.0`, we use a placeholder `_._` file to tell NuGet no asset should be picked (since we don't need a native asset on this TFM).

Reply:
Ah, I missed the `rid` section, which covers the bitness of the architecture

👍